scripts/Dockerfile.test

```diff
@@ -41,6 +42,7 @@ RUN dnf -y install \
     && mkdir /opt/commitlint && pushd /opt/commitlint \
     && npm init -y \
     && npm install @commitlint/cli \
+    && dnf install python3-prettytable \
```

Comment:
better add this to the list where pylint is installed too. This will cause the package metadata to be downloaded again (because it is after `dnf clean all` above)

Reply:
Oh sorry, I missed that.